packages/core/src/MatchMaker.ts

```diff
@@ -195,7 +195,7 @@ export async function findOneRoomAvailable(roomName: string, clientOptions: Clie
       roomQuery.sort(handler.sortOptions);
     }
 
-    return await roomQuery;
```

Comment:
if the function is awaited by the caller, then makes more sense to just remove the `async` and not await here